README.md

```diff
@@ -1,41 +1,87 @@
+- [ ] Replace `https://github.com/ably-labs/ably-labs-template-repo/issues` with the actual link of the repo in the [CONTRIBUTING.md](CONTRIBUTING.md) file.
+- [ ] Update this README so it provides enough information for people to understand how it works, how to run it locally and how it can be deployed to the cloud (see [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes)).
+- [ ] Add a GitHub workflow to build/test/deploy your application. Use the [Ably Control API GitHub action](https://github.com/ably-labs/ably-control-api-action) to avoid creating Ably apps/API keys manually.
+- [ ] Add this repository to the [selected repositories in the Ably Labs org](https://github.com/organizations/ably-labs/settings/actions) that are allowed to run GitHub Actions.
 
 Once you're done, remove this section from the README. Good luck! 💪
 
----
 
-![Place eye candy header image here](https://placekitten.com/640/360)
 
-*// Place eye candy header image here ⬆️*
+# Netlify Identity & JWT authentication
+
+
+![Netlify + JWT + Ably](./assets/netlify-ably-jwt.png)
 
 ## Description
 
 // Explanation of the contents of the repository. Describe the use case.
 
-## Tech stack
 
-![A high level architecture diagram tells more than a 1000 words](https://placekitten.com/480/240)
 
-*// A high level architecture diagram tells more than a 1000 words ⬆️*
 
-The project uses the following components:
+## What you'll need
+
+1. An Ably API Key, sign up for a [free account](https://ably.com/sign-up)
+1. A [Netlify](https://netlify.com) account to host and manage your registered users.
+1. A [Github](https://github.com) account is required to link Netlify to the repository.
+
+# Overview of architecture
+
+This is the overview of our app. Our user experience requires a new user to register and confirm their email address to activate themselves with our app before they can log-in. When they login we validate them with Netlify Identity and check that they have not been flagged as Banned. Bad actors are not issued with a JWT token. Valid users are issued a token, to authenticate with Ably, and the authentication URL carries the user's unique ID.
+
+![](./assets/user-experience-netily-functions.png)
+
+The Netlify Identity allows us to administer users by editing metadata associated with their account, flagging a bad actor is a matter of assigning them a role via the Netlify dashboard.
+
+## Initial steps
+1. Fork this repository
+1. Use Netlify dashboard to create a new project, and select your fork as the source.
+1. Add the environment variables to the new Netlify app from the site settings section, under `Build & deploy`
+1. Deploy the website, visit the homepage and sign-up as a new user.
```

Comment:
After I performed these steps, when I visit my deployed site, I just get a 404.

Here's what I see in Netlify:

<img width="1299" alt="Screenshot 2022-03-03 at 17 15 27" src="https://user-images.githubusercontent.com/488515/156616416-42732d05-b854-4005-897e-b95fb7f5b81c.png">

And here's what I see when I click the link to the site:

<img width="1300" alt="Screenshot 2022-03-03 at 17 16 30" src="https://user-images.githubusercontent.com/488515/156616578-5620bc3c-b5d7-48c7-a3b8-c85cf186fbd4.png">

Is there something more I need to do?

Reply:
damn. it is probably because the `netlify.toml` is not pointing at the webroot `public/`
thanks i should completely delete see if i can replicate this.

Reply:
Ah thanks, I pulled that change into my branch and it works now 👍

Reply:
But then next issue, when I click "Sign up", I get an error:

<img width="597" alt="Screenshot 2022-03-04 at 19 07 19" src="https://user-images.githubusercontent.com/488515/156826313-f525ec1a-5f39-44ba-b27c-c99a89db094e.png">

Reply:
Ah so I had to go back to my app in Netlify, go to the Identity tab and click "Enable Identity", probably worth adding that as a step in the instructions